src/test/suite/fixtures/recorded/selectionTypes/clearCar.yml

```diff
@@ -0,0 +1,23 @@
+languageId: plaintext
+command:
+  spokenForm: clear car
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: character}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: aaaBbbCcc
+  selections:
+    - anchor: {line: 0, character: 4}
+      active: {line: 0, character: 4}
+  marks: {}
+finalState:
+  documentContents: aaabbCcc
```

Comment:
huh.  I think I'd go right here.  That's what we do for surrounding pair

Reply:
I follow the logic we use with token. There we prefer left.